4-4-3.py

```diff
@@ -0,0 +1,39 @@
+'''
+След матрицы
+
+Следом квадратной матрицы называется сумма элементов главной диагонали. Напишите программу, которая выводит след заданной квадратной матрицы.
```

Comment:
And let's start to format such descriptions in order for them to fit into 140 characters - it's not convenient to read them as you must scroll right all the time.